quarkc/helpers.py

```diff
@@ -38,6 +38,10 @@ def __init__(self, loc, warning):
     def __str__(self):
         return "%s: warning: %s" % (lineinfo(self.loc),  self.warning)
 
+@dispatch(AST, object)
```

Comment:
Docstring for `get_field` in general would be helpful, and explanation for what this particular variant does.